### ISSUE TITLE
search: both searchbars search "wider ecosystem"

### ISSUE THREAD

**Issue by yarikoptic** [ReproNim/repronim.github.io#106]
I thought I had checked locally but I guess - not enough.

I guess it is just a matter of the 2nd script to overload the first one in that added search.html:
```
            		<div class="row">
            			<div class="col-md-12">
            			  <h2>this website:</h2>
            			  <script async src="https://cse.google.com/cse.js?cx=011585779972124617449:lngfk9yjbrv"></script>
            			  <div class="gcse-search"></div>
            			</div>
            		</div>
            		<div class="row">
            			<div class="col-md-12">
            			  <h2>larger ReproNim ecosystem:</h2>
            			  <script async src="https://cse.google.com/cse.js?cx=011585779972124617449:nhlyoiogpg7"></script>
            			  <div class="gcse-search"></div>
            			</div>
```
apparently it is a known trouble, and some solutions were suggested: https://stackoverflow.com/questions/33087411/multiple-gcses-on-page-at-one-time/42427119#answer-42427119  so might try later or would welcome contributions ;)

**Comment by dnkennedy** [comment]
So, what indeed is "larger ReproNim ecosystem:" supposed to be?  It looks the same as the "this site".

Should we drop the second search until the above 'known trouble' is resolved?

**Comment by yarikoptic** [comment]
as I have even provided pointers on how to fix, I hoped that someone closer to the website administration would resolve it following the instructions...  ok -- I will resolve it properly today

**Comment by yarikoptic** [comment]
pushed the fix in f1ebc52ca57e7e5b8ae5609b73e6fb262b77cdc4 , added (invited?) you @dnkennedy to admins for that CSE setup so you could add more websites into "ecosystem". Let's close if after deployment it does the right thing

**Comment by yarikoptic** [comment]
works as intended, closing
